.github/workflows/CI.yaml

```diff
@@ -229,6 +229,12 @@ jobs:
         sleep 30
         make integration-test
 
+    - name: Apply extra config options
```

Comment:
I had it there originally but the change kept getting reverted. I believe it's because of the checkout that happens right after the `Start Quay` step.